docs/reference/management_commands.md

```diff
@@ -162,5 +162,5 @@ This does not remove unused rendition images, this can be done by clearing the f
 
 Options:
 
--   **--purge-only** :
+-   **-- --purge-only** :
```

Comment:
This doesn't look right. As shown in your screenshot and [in the rendered preview](https://wagtail--11694.org.readthedocs.build/en/11694/reference/management_commands.html#wagtail-update-image-renditions) there is a space between the dashes that shouldn't be there.

The fact that the code here originally already uses a double dash means that it's probably because Markdown doesn't play well with the bullet point followed by the double dash. We should wrap this in inline code instead, e.g.

```suggestion
- `--purge-only`:
```